src/yadism/coefficient_functions/__init__.py

```diff
@@ -43,58 +43,47 @@ def __init__(self, esf):
         self.obs_name = esf.info.obs_name
         self.nf = esf.info.threshold.nf(esf.Q2)
         self.target = esf.info.target
+        self.scheme = esf.info.scheme
 
     def collect(self):
         comps = []
-
         family = self.obs_name.flavor_family
-
         # Adding light component
         if family in ["light", "total"]:
             comps.append(self.light_component())
-        if family == "heavy":
-            #  the only case in which an heavy contribution is not present in those
-            #  accounted for in total, it's whene heavy already became heavylight
+        if family == "heavy" and self.scheme != "FFN0":
```

Comment:
Most likely, we should discuss this part in more details.

Reply:
to be more specific: when we need to compute F2c in FFNS4 and FFN04 they are just the same: heavylight

Reply:
Ah yes you may be right. The reason I left it out was that in the current master implementation it is only evaluated once in the FONLL scheme so I figured any cancellation of this term would be in the `*diff` function and thus including this in FFN0 would result in a double counting. However, this term should probably cancel between the contributions in three flavor scheme (in which I don't believe it contributes, but still the exception for FFN0 shouldn't be here) and then result in a net contribution for the 4 flavor scheme.